src/PeoplePage.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { EnumPersonType } from './types/EnumPersonType';
+import { PersonLink } from './PersonLink';
+import { Person } from './types';
+
+interface PeoplePageProps {
+  usersArr: Person[];
+}
```

Comment:
Consider renaming 'usersArr' to a more descriptive name like 'people' or 'persons'. This would make the code more self-explanatory.